mysql-client-tests/node/index.js

```diff
@@ -15,41 +15,40 @@ var config = {
 
 class Database {
     constructor( config ) {
-        this.connection = mysql.createConnection( config );
+				this.connection = mysql.createConnection( config );
+				this.connection.connect();
     }
 
     query( sql, args ) {
         return new Promise( ( resolve, reject ) => {
             this.connection.query( sql, args, ( err, rows ) => {
                 if ( err )
                     return reject( err );
-                resolve( rows );
+                return resolve( rows );
             } );
         } );
     }
     close() {
-        return new Promise( ( resolve, reject ) => {
-            this.connection.end( err => {
-                if ( err )
-                    return reject( err );
-		process.exit(0);
-                resolve();
-            } );
-        } );
+      this.connection.end(err => {
+				if (err) {
+					console.error(err)
+				} else {
+					console.log("db connection closed")
+				}
+			})
```

Comment:
My changes originated from this app hanging and not exiting properly in the docker container... locally it seemed to exit without hanging, but not when dockerized... just refactored this to match what was in the npm example.